### DISCUSSION TITLE
When will the complete training code be released?

### DISCUSSION THREAD

**Discussion by Reidvoss** [PixArt-alpha/PixArt-alpha#4] [Q&A] [answered]
When will the complete training code be released?

**Comment by lawrence-cj** [comment, marked as answer]
You may access the whole training code now.

**Reply by levi** [reply]
Will you also release the LLaVa based captioning pipeline?

**Reply by lawrence-cj** [reply]
The pipeline is discussed in our [paper,](https://arxiv.org/abs/2310.00426) and the process is easy and simple. Btw, we will release the LLaVA-captioned SAM dataset in the near future.

**Reply by alfredplpl** [reply]
Great work! Btw, we need LLaVA-captioned SAM dataset to train model by SA-1B. At [the Meta page](https://ai.meta.com/datasets/segment-anything-downloads/), the original SA-1B contains only jpg and json. We need the conversion code?

**Reply by yujincheng08** [reply]
we will release the LLaVA-captioned SAM dataset

**Reply by Reidvoss** [reply]
Thank you！！！  I have another greedy request. I would like to ask about the opening hours of weights.

**Comment by lawrence-cj** [comment]
We will release the caption keeping the original jpg and json name unchanged. On Oct 28, 2023, at 17:45, Alfred Increment ***@***.***> wrote:﻿
Great work! Btw, we need LLaVA-captioned SAM dataset to train model by SAM. At the Meta page, it contains only jpg and json. We need the conversion code?

**Reply by alfredplpl** [reply]
Thank you so much. I will wait for the release.

**Reply by adhikjoshi** [reply]
When diffusers pipeline will be available?

**Reply by lawrence-cj** [reply]
Collaborating with Diffusers & Huggingface team. Plz stay tuned.
